code/client/clrcore/External/Screen.cs

```diff
@@ -327,7 +327,28 @@ private static PointF WorldToScreen(Vector3 position, float screenWidth, float s
 			}
 
 			return new PointF(pointX * screenWidth, pointY * screenHeight);
-		}
+		}
+
+		/// <summary>
+		/// Draws a text on the screen
+		/// </summary>
+		/// <param name="text">The text do draw</param>
+		/// <param name="x">The x offset between 0.0 and 1.0</param>
+		/// <param name="y">The y offset between 0.0 and 1.0</param>
+		/// <param name="scaleX">The x scale of the text</param>
+		/// <param name="scaleY">The y scale of the text</param>
+		/// <param name="color">The text color</param>
+		/// <param name="font">The font to use for the text</param>
+		public static void DrawText(string text, float x, float y, float scaleX, float scaleY, Color color, int font)

```

Comment:
Why? There's a `Text` class exactly for this purpose. https://github.com/citizenfx/fivem/blob/master/code/client/clrcore/External/Text.cs

Reply:
Oops, I didn't see that one.